### ISSUE TITLE
systemd required

### ISSUE THREAD

**Issue by vlp-git** [probonopd/go-appimage#209]
Hello,

I have tried to launch it on Alpine but unfortunately, it seems that systemd is required ?
Or did I miised smth to configure ?

Thanks !!

**Comment by probonopd** [comment]
Yes, appimaged is supported on a certain set of Live ISOs which all use systemd, so systemd currently is a requirement.

systemd is not really central to how appimaged works, could be removed easily.

**Comment by SlySven** [comment]
Is this still the case as it is a blocker for using AppImages on the **[Devuan](https://www.devuan.org/)** GNU/Linux distribution... :cry:

**Comment by vadi2** [comment]
appimaged is optional to using AppImages, so if I have it right, it shouldn't be?

**Comment by probonopd** [comment]
Correct, appimaged is entirely optional.
